test/langtools/tools/javac/7166455/CheckACC_STRICTFlagOnclinitTest.java

```diff
@@ -25,98 +25,84 @@
  * @test
  * @bug 7166455
  * @summary javac doesn't set ACC_STRICT bit on <clinit> for strictfp class
- * @modules jdk.jdeps/com.sun.tools.classfile
- * @compile -source 16 -target 16 CheckACC_STRICTFlagOnclinitTest.java
```

Comment:
This compilation cannot be removed and conversion of this test must wait until Class-File API leaves the preview.